src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -56,6 +58,21 @@ const webhookRouter = (context: WebhookContext) => {
 				GithubWebhookMiddleware(issueWebhookHandler)(context);
 			}
 			break;
+		case "pull_request":
+			if (context.action === "opened" || context.action === "closed"
+				|| context.action === "reopened" || context.action === "edited") {
+				GithubWebhookMiddleware(pullRequestWebhookHandler)(context);
+			}
+			break;
+		case "pull_request_review":
+			GithubWebhookMiddleware(pullRequestWebhookHandler)(context);
+			break;
+		case "create":
+			GithubWebhookMiddleware(createBranchWebhookHandler)(context);
+			break;
+		case "delete":
+			GithubWebhookMiddleware(deleteBranchWebhookHandler)(context);
+			break;
 	}
```

Comment:
Is it worth logging out on `default:` the skipped webhooks? Might be more noise than its worth, but as I understand we subscribe to webhooks in the app permissions so we should only receive one we want?

Reply:
We receive webhook events only for those that we subscribe in app permission but there are certain event actions that we're ignoring. It will more noise in logs if we start logging it.